apps/notifications/src/panel/templates/gridicons.jsx

```diff
@@ -1,354 +1,349 @@
 import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
 import PropTypes from 'prop-types';
-import { Component } from 'react';
-
-export default class extends Component {
-	static displayName = 'Gridicons';
-
-	static propTypes = {
-		icon: PropTypes.string.isRequired,
-		size: PropTypes.number,
-		onClick: PropTypes.func,
+import { forwardRef } from 'react';
+
+const Gridicons = forwardRef( ( { icon, size = 24, onClick }, ref ) => {
+	const translate = useTranslate();
+	const gridIcon = `gridicons-${ icon }`;
```

Comment:
```suggestion
	const gridIconClass = `gridicons-${ icon }`;
```

Reply:
I changed to `gridiconClass`: https://github.com/Automattic/wp-calypso/pull/97784/commits/723354b50b6ff371f91a1bde79cb35c90247b717 It looks like we refer to it as one word ("gridicon") in most places, though I agree it looks weird.